frontend/public/locales/en/personal-information.json

```diff
@@ -78,27 +81,28 @@
   },
   "mailing-address": {
     "edit": {
-      "page-title": "Change mailing address",
-      "subtitle": "Please update your mailing address below.",
+      "page-title": "Mailing address",
       "breadcrumbs": {
         "personal-information": "Personal information",
-        "mailing-address-change": "Change mailing address"
+        "mailing-address-change": "Mailing address"
       },
-      "copy-home-address": "Copy home address",
-      "copy-home-address-note": "We have the following home address on file, which will be used as your mailing address:",
       "field": {
         "address": "Address",
-        "city": "City",
-        "province": "Province/State",
-        "postal-code": "Postal code",
-        "country": "Country"
-      },
+        "address-note": "Include street number and name.",
+        "apartment": "Apartment, suite, etc. (opetional)",
```

Comment:
```suggestion
        "apartment": "Apartment, suite, etc. (optional)",
```

Reply:
Do we need `(optional)`?

Reply:
(optional) is in the Figma i am working on

Reply:
removed